File: Data Structures & Algorithms/two-integer-sum/submission-1.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector<int> Result;
        for(int i=0;i<size(nums);i++){
            int remain = target-nums[i];
            for(int j=i+1;j<size(nums);j++){
                if(nums[j] == remain){
                    Result.push_back(i);
                    Result.push_back(j);
                    return Result;
                }
            }


        }
        
    }
};

File: Data Structures & Algorithms/max-consecutive-ones/submission-1.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        result = sum = 0
        for i in range(len(nums)):
            if(nums[i] == 1):
                sum +=1
            else:
                result = max(result , sum)
                sum = 0
        
        return max(result, sum)
            

        